Makefile

```diff
@@ -41,12 +41,24 @@ dbdiagram: #- Generate database diagram image
 
 test: test-server test-client #- Run the server and client test suite
 
+test-ci: test-server test-client-ci #- Run the server and client test suite in CI mode
+
 test-server: #- Run the server test suite
 	${bin}pytest
 
-test-client: #- Run the client test suite
+test-client: test-client-unit test-client-integration #- Run the client's unit and integration tests
```

Comment:
[Question terminologie]

À mon sens Cypress c'est un outil de "tests end-to-end (e2e)", puisqu'on démarre un système complet en simulant le comportement utilisateur. C'est d'ailleurs sur leur site : "JavaScript End to End Testing Framework"

Quid donc d'utiliser ce vocable et dire par ex `test-e2e` plutôt que `test-integration` ?

(Un test d'intégration côté front, j'imaginerais plutôt un test qui s'assure que le front réagit correctement à un événement qui appelle l'API, en mockant l'endpoint en question. Non ?)